src/core/wallet/services/certificateService/certificateService.ts

```diff
@@ -109,12 +109,12 @@ export class CertificateService {
       }));
   }
 
-  private customRequestTokenFactory = (certificateId, passphrase) => {
+  private customRequestTokenFactory = async (certificateId, passphrase) => {
     const temporaryWallet = this.configurationService
       .walletFactory()
       .fromPassPhrase(passphrase);
 
-    const proof = this.utils.signProofForRequestToken(
+    const proof = await this.utils.signProofForRequestToken(
```

Comment:
signProofForRequestToken shouldn't be asynchronous since web3.eth.accounts.sign is not.